lib/neat_and_clean_calendar_event.dart

```diff
@@ -27,5 +28,6 @@ class NeatCleanCalendarEvent {
       this.isAllDay = false,
       this.isMultiDay = false,
       this.isDone = false,
-      multiDaySegement});
+      multiDaySegement,
+      this.metadata});
```

Comment:
I don't know what you plan with this metadata property. As it is not related to the topic of the pull request I'd like you to remove it from the class. Maybe you can file another pull request that includes this property, if it is clear, what to use it for.